woocommerce/woocommerce-ios
# Shipping Labels: Ship items in original packaging

Part of #4717 

# Description
This PR adds implementation to move items to individual packages using their original packaging. It also includes validation for original packaging, which requires valid dimensions for individual items.

# Changes
- Packages view model:
   - Handles moving an item to an individual package using original packaging.
   - Updates validation for each package based on new validation variable.
- Single package view model:
   - Adds new parameter to initializer to determine if package is using original packing of its item.
   - Configures displayed dimensions for the original package
   - Updates validation to check for both total weight and dimensions if the package uses original packaging.
- Singe package view:
   - Adds new rows for original package
   - Adds validation error for original package 

# Demo
| Valid dimensions | Invalid dimensions |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/134854534-5237b5c3-5cab-47bd-b042-59cb2ac6c532.gif" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/134854709-08a49c7f-162c-459b-a4a3-acb6dbab1614.png" width=400 /> |

# Testing steps
1. Make sure that your test store has WCShip plugin installed and activated, with packages configured in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping.
2. Select an order eligible for creating shipping label. The order item list should contain a product with valid dimensions configured for its packaging.
3. Select Create Shipping Label and configure Ship From and Ship To.
4. Proceed to configure Package Details. Select the product with valid dimensions and tap Move > select Ship in original packaging.
5. Notice that the item is displayed in another package, which shows a new row: Original packaging. Correct dimensions are displayed in another row: Item dimensions.
6. Repeat steps 2-5 for another order that contains a product with invalid dimensions. Notice that the invalid dimensions are highlight in red color, with a validation error underneath the Item dimensions row, explaining the error. Done button should be disabled.

Update release notes:
